### DISCUSSION TITLE
[Q] Remove some element from selected element

### DISCUSSION THREAD

**Discussion by fer-ri** [MontFerret/ferret#696] [Q&A] [answered]
Hi

Let's say I've selected content with this

```
LET content = ELEMENT(doc, 'article.main-content')
```

And inside that `content` there are inline `style` and `script` tags. How to remove these tags from `content`?

I've tried to search inside docs and example folder but still not found

Thanks

**Comment by ziflex** [comment, marked as answer]
Hey,

There is no direct API that would allow doing so, but you can use the ``SET_INNER_HTML`` function to override the content.
The query would be:

```
LET content = ELEMENT(doc, 'article.main-content')
LET strippedHTML = REGEX_REPLACE(stripped.innerHTML,  expression, "")

SET_INNER_HTML(content, strippedHTML)
```

**Reply by fer-ri** [reply]
Awesome. Thanks :+1:
